spec/index.adoc

```diff
@@ -1743,6 +1743,12 @@ are returned, i.e., transactions with fewer than this number of
 confirmations are not considered. In other words, if the number of confirmations is `c`, an output
 is returned if it occurred in a transaction with at least `c` confirmations
 and there is no transaction that spends the same output with at least `c` confirmations.
+There is an upper bound of 144 on the minimum number of confirmations because only the 144 most recent
+blocks are cached and it is not possible to return the correct result based on the UTXO set.
```

Comment:
```suggestion
There is an upper bound of 144 on the minimum number of confirmations.
```

1) I don't think we need/should provide technical justifications or implementation details as part of the spec, so I'd lean towards removing that part.

2) The upper bound we choose doesn't have to be related to the stability threshold we have internally. Can we not start with a more conservative number (e.g. 10 or 20)? That can help us do some optimizations, and we can always expand that limit later.

3) If I understand correctly, I don't think we can always serve requests with 144 confirmations. In cases where we have forks, for example, we may not have enough information and return a `MinConfirmationsTooLarge` error. If that's correct, then maybe worth mentioning that this number is also not a strict lower bound?